libs/stylelint-config/package.json

```diff
@@ -15,18 +15,17 @@
   "dependencies": {
     "postcss-scss": "^4.0.9",
     "stylelint-config-property-sort-order-smacss": "^10.0.0",
-    "@stylistic/stylelint-plugin": "^2.1.2",
+    "@stylistic/stylelint-plugin": "^3.0.1",
     "stylelint-use-nesting": "^5.1.1",
     "stylelint-declaration-block-no-ignored-properties": "^2.8.0",
     "stylelint-file-max-lines": "^1.0.0",
-    "stylelint-scss": "^6.1.0",
+    "stylelint-scss": "^6.5.1",
     "stylelint-group-selectors": "^1.0.10"
   },
   "peerDependencies": {
-    "stylelint": "^16.5.0"
+    "stylelint": "^16.8.2"
```

Comment:
As usual, I reviewed the Stylelint releases to see what's been fixed and added since version 16.2.1 (the version before Tamara's update), including versions 16.5.0 and newer releases. I extended the selector-max-compound-selectors rule with ignoreSelectors and added the no-unknown-custom-media rule introduced in 16.4.0. All other changes were fixes. In the meantime, latest Stylelint version is released (16.9.0), I'll update to that as well."

Reply:
For the @stylistic/stylelint-plugin, I reviewed the releases from version 2.1.0 and updated to 3.0.1 (which I confirmed is compatible with Stylelint 16.8.2). The only new rule is named-grid-areas-alignment for grid-template-areas, but since we don't use that feature, I thought we might not need it